test-suite/misc/votour.sh

```diff
@@ -0,0 +1,3 @@
+command -v "${BIN}votour" || { echo "Missing votour"; exit 1; }
+
+"${BIN}votour" ../theories/Init/Logic.vo < /dev/null
```

Comment:
should be something stupid like `$($coqc -where)/theories/Init/Logic.vo`
Perhaps easier would be to compile a dummy file and run votour on that vo, that way you control where the vo is.